src/test/oauth/hubspot-oauth.steps.ts

```diff
@@ -0,0 +1,50 @@
+import { loadFeature, defineFeature, DefineStepFunction } from 'jest-cucumber';
+import { getHubspotAccessToken } from '../../routes/oath';
+require('dotenv').config();
+
```

Comment:
As per Amanda's feedback today, adding a short file comment on this and the SaaSquatch oauth tests could be helpful :) Also I don't think this affects anything but I think these are technically integration tests because they interact with the external Hubspot and SaaSquatch apis rather than mocking this behaviour. Otherwise looks good!